aiconfig-docs/docs/getting-started.md

```diff
@@ -258,11 +274,10 @@ Let's run this with AIConfig:
 Replace `config.run` above with this:
 
 ```python
-inference_options = InferenceOptions(stream=True)
```

Comment:
Don't we still need this? Or is this set earlier already?

Reply:
It is set earlier. User just needs to update the aiconfig.run() call